LICENSE.LGPL

```diff
@@ -0,0 +1,458 @@
+GNU LESSER GENERAL PUBLIC LICENSE
+             Version 2.1, February 1999
```

Comment:
I wonder if we should include 2.1 or 2.0 text when the declaration is "SPDX-License-Identifier: LGPL-2.0-or-later". I think both are correct though, so this is fine.

Reply:
Often considered almost the same thing but since it's 2.0 or later let's switch this to the older version.